ddtrace/internal/writer.py

```diff
@@ -210,7 +210,9 @@ def __init__(
         self.agent_url = agent_url
         self._buffer_size = buffer_size
         self._max_payload_size = max_payload_size
-        self._buffer = TraceBuffer(max_size=self._buffer_size, max_item_size=self._max_payload_size)
+        self._buffer = TraceBuffer(
+            max_size=self._buffer_size, max_item_size=self._max_payload_size
+        )  # type: ignore[call-arg]
```

Comment:
Why do we need this ignore here?

Sounds like we shouldn't be having any kind of error here?

Reply:
Our `attr` is vendored, which mypy doesn't recognize 😦

Reply:
hmm :(